docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,22 @@
+# Tutorial: deploy the gateway API integrator charm
+
+This tutorial will walk you through deploying the gateway-api-integrator charm
+
+## Prerequisites
+* A kubernetes cluster with a gateway controller installed.
+* A host machine with juju version 3.3 or above.
+
+## Deploy
+1. Deploy and configure the charm
+```
+juju deploy gateway-api-integrator
+juju config gateway-api-integrator external-hostname=ingress.internal
+```
+2. Deploy a TLS provider
+```
+juju deploy self-signed-certificates
+```
+3. Integrate the gateway-api-integrator charm with the TLS provider 
```

Comment:
This could be its own section, to be consistent with the statements in the "What you'll do" section. Perhaps deploying the self-signed-certificates charm goes under this section as well, but it might make more sense to keep the deployment commands in the same section.

Reply:
Added a separate section for this.